src/views/Question.vue

```diff
@@ -189,15 +193,14 @@ watch:{
   cursor: pointer;
 }
 
-.button-disabled {
+.submit-button[disabled] {
   height: 52px;
   width: 52px;
   opacity: 0.5;
   background-color: #4E4D86;
   color: white;
   border-radius: 10000px;
   margin-top: 60px;
```

Comment:
At this point, the styles for `[disabled]` don't have to be duplicated again since it'll inherit the original styles from the `button` itself. For example, don't have to specify `height: 52px;` again when the button by default already has `height: 52px;`.